packages/dashboards/addon/routes/dashboards/dashboard/widgets/add.js

```diff
@@ -31,10 +31,8 @@ export default Route.extend({
         widget.set('dashboard', dashboard);
 
         return widget.save().then(({ id }) => {
-          let layout = get(dashboard, 'presentation.layout'),
-            newLayout = this._addToLayout(layout, Number(id));
-
-          set(dashboard, 'presentation.layout', newLayout);
```

Comment:
👍 Looking into this, we no longer need this b/c we moved away from `layout` being an array to a `fragment` (which has its own observer).